azurerm/internal/services/web/function_app_resource_test.go

```diff
@@ -901,6 +901,43 @@ func TestAccFunctionApp_scm(t *testing.T) {
 	})
 }
 
+func TestAccFunctionApp_clientCertMode(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_function_app", "test")
+	r := FunctionAppResource{}
+
+	data.ResourceTest(t, r, []resource.TestStep{
+		{
+			Config: r.basic(data),
+			Check: resource.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+				check.That(data.ResourceName).Key("client_cert_mode").HasValue("Ignore"),
```

Comment:
which means this can become:

```suggestion
				check.That(data.ResourceName).Key("client_cert_mode").HasValue(""),
```